ArjunaJTA/jdbc/tests/classes/com/hp/mwtests/ts/jdbc/basic/ConnectionImpleTest.java

```diff
@@ -125,13 +124,14 @@ public void beforeCompletion() { }
             public void afterCompletion(int status) {
                 try {
                     connectionToTest.isClosed();
+                    connectionToTest.close();
```

Comment:
Did you have change to review this yet? I wonder about if adding the .close would also be useful if we in the future added pooling to ConnectionImpl. By not calling .close we would leak the connection not just from ConnectionManager but also this hypothetical pool.

Thanks!